bin/avhrr_l0.rb

```diff
@@ -23,11 +22,15 @@ def execute
       sourcefile = uncompress(sourcefile)
 
       year = Time.now.strftime('%Y')
-      command = "hrptin #{conf['opts']} tape_device=./#{sourcefile} pass_year=#{year} ."
-      shell_out!(command)
-
-      conf['save'].each do |pattern|
-        copy_output(output, pattern)
+      begin
+        command = "hrptin #{conf['opts']} tape_device=./#{sourcefile} pass_year=#{year} ."
+        shell_out!(command)
```

Comment:
You should be able to just change this to the `shell_out(command)` and it will still log the error as having happened but won't throw an exception